dev/src/Command/SplitCommand.php

```diff
@@ -157,9 +163,11 @@ protected function execute(InputInterface $input, OutputInterface $output)
         );
 
         $releaseNotes = new ReleaseNotes($changelog);
+        $branchToPush = null;
 
         if ($componentId = $input->getOption('component')) {
-            $components = [new Component($componentId)];
+            $branchToPush = $input->getOption('component-branch');
```

Comment:
There's no reason this can only happen for a single component - if we push changes to multiple components at once (via a tag), we should be able to supply the `component-branch` as well. 

In fact, the workflow we've talked about makes the most sense when _not_ passing in a `component` option, as we'd want to use the tag to determine which components to release, e.g.:

./google-cloud split googleapis/google-cloud-php v0.333.1 --component-branch v1-main